tests/fixtures/toml/redirects.toml

```diff
@@ -47,6 +47,19 @@ source = "**/{*}.{ttf,otf,woff}"
 destination = "http://localhost/new-fonts/$2.woff"
 kind = 302
 
+# Glob groups ranges 1
+[[advanced.redirects]]
+source = "/[0-9]/[a-z]/{*}/"
+destination = "http://localhost/new-range/$1/"
+kind = 301
+
+# Glob groups ranges 2
+[[advanced.redirects]]
+source = "/crop-{[!0-9]}/{*}.{png,gif}"
```

Comment:
not sure how useful this is. I am struggling to get something like this working where more than one character can be matched. `[!0-9]*` results in a `[^0-9].*` regex because `globset` replaces `*` with `.*`.

Reply:
I understand. It was just an idea, but we could skip it now if it gives us headaches.